web/src/admin/AdminPoster.js

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import { withRouter, Link } from "react-router-dom";
+import * as MemberBackend from "../backend/MemberBackend.js";
+import * as PosterBackend from "../backend/PosterBackend"
+import * as Setting from "../Setting";
+import PageColumn from "../main/PageColumn";
+import Avatar from "../Avatar";
+import Collapse, { Panel } from "rc-collapse";
+import Select2 from "react-select2-wrapper";
+import i18next from "i18next";
+
+class AdminMember extends React.Component {
+    constructor(props) {
+        super(props);
+        this.adver=React.createRef();
+        this.links=React.createRef();
+        this.p_link=React.createRef();
+        this.state = {
+            classes: props,
+            message: "",
+            form: {},
+        };
+
+        this.state.url = `/admin/poster`;
+    }
+
+    componentDidMount() {
```

Comment:
For the admin page, I think you should read the current advertisement and display it.